data/json/items/comestibles/bread.json

```diff
@@ -224,16 +224,15 @@
     "material": [ "fruit", "wheat" ],
     "primary_material": "wheat",
     "spoils_in": "7 days",
-    "relative": { "calories": 35, "fun": 3, "quench": 2, "vitamins": [ [ "vitC", "20 mg" ] ] },
-    "vitamins": [ [ "wheat_allergen", 1 ], [ "fruit_allergen", 1 ] ]
```

Comment:
I understand that the relative vitamins doesn't work, but is it correct to remove allergens like that? We get copy-from'd from toast, which doesn't have fruit_allergen, so this line is neccessary to add it back, right?

Should it become ` "extend": { "vitamins": [["fruit_allergen", 1]] }` like you did for protein shake? (i don't know if that one actually works either)

(same concern for other allergen removals)